### ISSUE TITLE
Possible problem with Project MMO

### ISSUE THREAD

**Issue by Dragom626** [ZsoltMolnarrr/BetterCombat#389]
**Minecraft version** - 1.20.1
**Mod loader** - Forge 47.3.5
**Mod version** - 1.8.5
**I am using the latest version available** - No

**Describe the bug**
I was fighting a Baby Skeleton from the mod "Mekanism Additions" using a sword from "Tetra" that uses the "Better Combat" attack and my game decided to crash. I was able to log back into my world and continue without any issue but I did receive a crashlog pointing to "Better Combat" and "PMMO". Currently unable to recreate but the steps I took before crashing are as follows.

**To Reproduce**
1. Found a burning(daytime) Baby Skeleton
2. Got shot by the Baby Skeleton
3. Engage Baby Skeleton in combat
4. Attack the Baby Skeleton at the same time as I get hit by an arrow
5. Both I and the Baby Skeleton take damage at the same time
6. Game crashed

**Expected behavior**
Fight the Baby Skeleton and defeat it without the game crashing.

**Screenshots or video recordings**
Do not have any screenshots or recordings.

**Additional context**
I don't know if this is a issue with either mod, an unrelated Java/Vanilla MC/Forge issue. If this is unrelated to Better Combat I have already reported it on PMMO's page also and request this be closed.

**Crash Log**
https://pastebin.com/UWCy1AR6

**Comment by Dragom626** [comment]
Additional crash logs here https://pastebin.com/RZPD8SGt and I posted this issue on the PMMO github to.

**Comment by ZsoltMolnarrr** [comment]
I do not see anything specific clue that Better Combat would be causing the problem.
I need more specifications for this issue, otherwise it is non-actionable for me. :(

**Comment by Dragom626** [comment]
@ZsoltMolnarrr Talking to the dev of PMMO it's most likely an issue on PMMO's side with how it handles NBT data. I did manage to fix my own issue to some extent(fixed the crash, failed to compile PMMO properly) so no more crashing but lang translations not getting applied properly(e.g. pmmo.skillname instead of the actual skill). Closing the issue here as the PMMO dev said the issue would probably happen with dyed armor being handled by PMMO.
